build/import/VisualStudio.XamlRules.targets

```diff
@@ -5,13 +5,30 @@
     <XamlPropertyRule>
       <Namespace>Microsoft.VisualStudio.ProjectSystem</Namespace>
       <RuleInjectionClassName>$(XamlPropertyRuleInjectionClassName)</RuleInjectionClassName>
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </XamlPropertyRule>
+    <XamlPropertyRuleNoCodeBehind>
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
```

Comment:
These are needed so that I can refer to the "CopyToOutputDirectory" metadata item without qualification (ie no need for DesignTimeTargetsFile.CopyToOutputDirectory);